samples/copyclean.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#    This file is part of python-registry.
+#    It tries to make a clean copy of hive file
+#    clean meaning without unused (not understood) data
+#
```

Comment:
what I define as junk is also cells and hbins that has never been used but still have non zero bytes in them. It is incorrect to call them junk if there is rests of real records in there, but that is not my experience so far.